stdlib/zipfile/_path/glob.pyi

```diff
@@ -0,0 +1,22 @@
+import sys
+from collections.abc import Iterator
+from re import Match
+
+if sys.version_info >= (3, 13):
+    class Translator:
+        def __init__(self, seps: str = ...) -> None: ...
+        def translate(self, pattern: str) -> str: ...
+        def extend(self, pattern: str) -> str: ...
+        def match_dirs(self, pattern: str) -> str: ...
+        def translate_core(self, pattern: str) -> str: ...
+        def replace(self, match: Match[str]) -> str: ...
+        def restrict_rglob(self, pattern: str) -> None: ...
+        def star_not_empty(self, pattern: str) -> str: ...
+
+else:
+    def translate(pattern: str) -> str: ...
+    def match_dirs(pattern: str) -> str: ...
+    def translate_core(pattern: str) -> str: ...
```

Comment:
Interesting. These two were added somewhere between 3.12.3 and 3.12.7.

Reply:
Oh yeah, I didn't notice that. Looks like there was a bug where zipflile globs would just ignore directories. `translate_core` wasn't really a new addition, it's the same as what `translate` was before `match_dirs` was added.